exporter/kineticaexporter/config.go

```diff
@@ -0,0 +1,253 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kineticaexporter // import
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configopaque"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	// The value of "type" key in configuration.
+	typeStr = "kinetica"
```

Comment:
When you run `make generate` it will generate a type string for you to use based on the metadata.yaml file.

Reply:
@am-kinetica This is the last thing to update.  We can remove this constant and use the value from `internal/metadata`

Reply:
Removed this with the generated value